### PR TITLE
feat: add Qwik example app

### PR DESCRIPTION
**Pull request by gioboa**
## 🧢 Checklist

- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged

## 📌 Resources

- [Security guidelines](https://github.com/nextauthjs/.github/blob/main/SECURITY.md)
- [Contributing guidelines](https://github.com/nextauthjs/.github/blob/main/CONTRIBUTING.md)
- [Code of conduct](https://github.com/nextauthjs/.github/blob/main/CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

File: apps/examples/qwik/package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "dev.debug": "node --inspect-brk ./node_modules/vite/bin/vite.js --mode ssr --force",
+    "fmt": "prettier --write .",
+    "fmt.check": "prettier --check .",
+    "lint": "eslint \"src/**/*.ts*\"",
+    "preview": "qwik build preview && vite preview --open",
+    "start": "vite --open --mode ssr",
+    "qwik": "qwik"
+  },
+  "devDependencies": {
+    "@auth/qwik": "workspace:*",
```
Comment (ThangHuuVu) [inline apps/examples/qwik/package.json:30]:
```suggestion
    "@auth/qwik": "latest",
```

File: apps/examples/qwik/src/routes/index.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          </div>
+          <a class="py-4 font-bold text-blue-600" href="/protected">
+            this protected route.
+          </a>
+        </>
+      ) : (
+        <div class="pt-6 text-2xl font-bold text-slate-800">
+          You can SignIn with GitHub in the header
+        </div>
+      )}
```
Comment (ThangHuuVu) [inline apps/examples/qwik/src/routes/index.tsx:34]: Let's remove this line, this example will be deployed and we will use https://authjs.dev/getting-started/deployment#auth_redirect_proxy_url to support many providers

**Review by ThangHuuVu — CHANGES_REQUESTED** [review]

File: apps/examples/qwik/src/routes/index.tsx (hunk 2 above)
Comment (gioboa) [reply apps/examples/qwik/src/routes/index.tsx:34]: Done

File: apps/examples/qwik/package.json (hunk 1 above)
Comment (gioboa) [reply apps/examples/qwik/package.json:30]: Done

**Review by ThangHuuVu — APPROVED** [review]

**Pull request merged**